core/sdk-core/src/main/java/software/amazon/awssdk/core/interceptor/Context.java

```diff
@@ -86,6 +106,9 @@ private Context() {}
     @ThreadSafe
     @SdkPublicApi
     public interface BeforeTransmission extends ModifyHttpRequest {
+        RequestBody requestBody();
+
+        AsyncRequestBody asyncRequestBody();
```

Comment:
Shouldn't this be in AfterMarshalling?

Reply:
Not if we have them as separate arguments in interceptors.